dashboards/amazon-workspaces/amazon-workspaces.json

```diff
@@ -0,0 +1,449 @@
+{
+    "name": "Amazon WorkSpaces",
+    "description": null,
+    "pages": [
+      {
+        "name": "Amazon WorkSpaces",
+        "description": null,
+        "widgets": [
+          {
+            "title": "Available WorkSpaces",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 3,
+              "height": 2
+            },
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT average(aws.workspaces.Available) AS 'Available WorkSpaces' WHERE aws.Namespace = 'AWS/WorkSpaces'"
```

Comment:
@rahul188 Is the value of aws.Namespace same for other users as well ?

Reply:
@mdumpati, addressed, updated the query.